### DISCUSSION TITLE
Welcome to IMDB-Trakt-Syncer Discussions!

### DISCUSSION THREAD

**Discussion by RileyXX** [RileyXX/IMDB-Trakt-Syncer#32] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

Before you post your Support question, have you tried solving the problem yourself:
1. Searching existing issues
2. Searching existing discussions

NOTE: Discussions are intended for community to community communication. Errors and Feature requests need to be reported as [issues](https://github.com/RileyXX/IMDB-Trakt-Syncer/issues/new/choose).

**Comment by simonlazopedraza** [comment]
Hello, is there any possibility to make the watched movies sync from imdb to trakt?

**Reply by RileyXX** [reply]
IMDB only supports watchlists, ratings and reviews/comments which can be synced. IMDB doesn't have any support for watch history so no syncing watch history would not be possible.

**Reply by MrN0b0dy98** [reply]
Maybe you can take a look at this [IMDB Check-ins](https://help.imdb.com/article/imdb/track-movies-tv/check-ins-faq/GG59ELYW45FMC7J3) "Check-ins allow you to share what you're watching or **previously watched**."

**Reply by simonlazopedraza** [reply]
Thank you!

**Reply by RileyXX** [reply]
@MrN0b0dy98 Thank you for the suggestion. I looked into IMDB Check-ins and this seems like a good feature to add. I've created an issue so you can monitor its progress: https://github.com/RileyXX/IMDB-Trakt-Syncer/issues/118

**Reply by RileyXX** [reply]
@simonlazopedraza @MrN0b0dy98 

Syncing Trakt watch history with IMDB Check-ins feature is added now in [v3.4.0](https://github.com/RileyXX/IMDB-Trakt-Syncer/releases/tag/v3.4.0).

```bash
python -m pip install IMDBTraktSyncer --upgrade
```

**Reply by aaronsewall** [reply]
I don't think IMDb check-ins are a perfect proxy for Trakt watch history since they are limited to 10,000 items, and Trakt watched history doesn't have this limit. My personal convention is that I rate things I've seen on IMDb with either a real rating, or I take the existing consensus rating and round it up to nearest whole number (e.g. a 7.2 gets an 8).

**Comment by TomGH77** [comment]
I have this error at step 5:
"trakt
OAuth error
Client authentication failed due to unknown client, no client authentication included, or unsupported authentication method."

Any ideas to fix it?

**Reply by RileyXX** [reply]
Apologies for the late reply. I don't get notifications from this discussion thread. Are you still having issues with this? There have been several updates since you posted this which might have fixed the problem you were having. 

If you're still having issues, feel free to open a bug report with all relevant error messages https://github.com/RileyXX/IMDB-Trakt-Syncer/issues/new/choose.
